### ISSUE TITLE
Do not prompt for upgrade if global switch is off

### ISSUE THREAD

**Issue by tyrchen** [ArcBlock/forge-js#375]
This is just a proposal. The requirement is that for automation scripts we don't want to upgrade forge-cli without explicitly doing so. We don't want to be prompted to enter yes or no. Is that possible something like this:

cat ~/.forge_clirc
{"prompt": false}

If the configure exists and the value is false forge cli won't do any upgrade check and prompt upon executing a sub command.

**Comment by wangshijun** [comment]
@tyrchen disable upgrading check landed in forge-cli v0.37.x, the global config file is stored at: `~/.forgerc.yml`, and the content maybe:

```
autoUpgrade: false
registry: https://registry.npm.taobao.org
mirror: http://arcblockcn.oss-cn-beijing.aliyuncs.com
moderatorSecretKey: -4WNJeYYg9txVW1FdaYQ5danLrx9B8pCAG53d2an-6GxVnDbuoXTXbfsZti7-0Tnry8XpRRD5siPWJEr5ngd5A
```

The `autoUpgrade: false` tells forge-cli not to check for updates.

For global non-interactive option, we need to do that in a new release.

**Comment by wangshijun** [comment]
close because forge-cli have resolved this.
